src/plugins/kibana_usage_collection/server/collectors/management/types.ts

```diff
@@ -12,6 +12,7 @@ export interface UsageStats {
    */
   'securitySolution:defaultIndex': string;
   'securitySolution:defaultThreatIndex': string;
+  'securitySolution:alertTags': string;
```

Comment:
Q: should this be `string[]`?

Reply:
I was thinking that, yeah, but I ran into a [test failing](https://buildkite.com/elastic/kibana-pull-request/builds/130830#01886862-9bce-44c9-bc72-927b7cd72e71) with an error message about [this type](https://github.com/elastic/kibana/pull/157786/files#diff-ca257f4b73b502a83cc8bdc63b0e8ff402b64e898360bdcc54d589cea1bd3f45) having to be a keyword and not an array, like the other `string[]` types in that file. This is consistent with the other string arrays in the file, but if that type works better, I can change it